addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1808,4 +1808,26 @@ options.registry.CoverProperties = options.Class.extend({
         this.$target[0].dataset.filterColor = activeFilterColor || '';
     },
 });
+
+/**
+ * Whether the section should be full-width (container-fluid) or use a classic container
+ */
+options.registry.SectionStretch = options.Class.extend({
+    /**
+     * Only shows this option if it changes the visual.
+     *
+     * @override
+     */
+    start: function () {
+        const $container = $('div', {class: 'container'}).insertAfter(this.$target);
```

Comment:
@sdegueldre The branch does not work because you put `'div'` instead of `'<div/>'` or `'<div>'` here :cry:  When you send a branch into review, be sure to retest everything :wink:

Reply:
What do you mean by "the branch does not work"? I did retest the functionality and it still worked after the change. Were some tests failing? I have however corrected the problem, just wondering how I could reproduce the issue for future reference.

Reply:
@sdegueldre It works now but it cannot work with `'div'` as it searches for a div in the second parameter you gave (a random object for him). So this line did nothing then sizeDifference was equal to NaN and thus the option was always displayed.